com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs

```diff
@@ -693,6 +705,15 @@ internal static void ResetAllHistoryRTHandleSystems(int width, int height)
                 }
             }
         }
+    
+        unsafe internal void UpdateScalesAndScreenSizesCB(ref ShaderVariablesGlobal cb)
+        {
+            cb._ScreenSize = screenSize;
+            cb._PostProcessScreenSize = postProcessScreenSize;
+            cb._RTHandleScale = RTHandles.rtHandleProperties.rtHandleScale;
+            cb._RTHandlePostProcessScale = RTHandles.rtHandleProperties.rtHandlePostProcessScale;
```

Comment:
We might need the _RTHandleScaleHistory  equivalent too as we create history for some PP passes too.

Reply:
Done